flatpak/org.nickvision.tubeconverter.json

```diff
@@ -171,6 +172,7 @@
                 }
             ]
         },
+        "libsecret/libsecret.json",
```

Comment:
Why couldn't you just put the libsecret module here....no need to keep it in another file

Reply:
Indeed, sorry, I first used libsecret from shared-modules and so placed it in separate folder (the same as with intltool), I will move it.